spotifyutils/playlists/__main__.py

```diff
@@ -0,0 +1,4 @@
+from spotifyutils.playlists import cli
```

Comment:
This is the way you can run a python script like:

```bash
python -m spotifyutils.playlists
```

Reply:
Confirming I understand, I run the python scrips and specify the auth parameters which then stores these parameters as env variables. Do I have that right?